x/gov/client/rest/rest.go

```diff
@@ -12,12 +12,13 @@ import (
 
 	"github.com/gorilla/mux"
 	"github.com/pkg/errors"
+	"github.com/cosmos/cosmos-sdk/x/gov/client"
 )
 
 // REST Variable names
 // nolint
 const (
-	RestProposalID     = "proposal-id"
+	RestProposalID     = "proposalId"
```

Comment:
let's use `snake_case` here

Reply:
Do you mean replace `proposalId` with `proposal_id`? That will be change many gov endpoints' name.

Reply:
ok np, let's leave it for later